x11-libs/wxGTK/wxGTK-3.0.4-r302.ebuild

```diff
@@ -85,6 +85,11 @@ src_prepare() {
 		-e "s:\(WX_SUBVERSION=\).*:\1${WXSUBVERSION}:" \
 		-e '/WX_VERSION_TAG=/ s:${WX_RELEASE}:3.0:' \
 		configure || die
+		# PCHes are unstable and are disabled in-tree where possible
+		# See bug #504204
+		# Commits 8c4774042b7fdfb08e525d8af4b7912f26a2fdce, fb809aeadee57ffa24591e60cfb41aecd4823090
```

Comment:
Wrong place

Reply:
Fixed in ed9c73deaedebd147462b51bab58e858277bfea0